tools/doc-generator/doclet/src/test/java/tests/smallrye/AllTypes.java

```diff
@@ -106,4 +108,9 @@ public interface AllTypes extends IntOne {
   /** Map of string to {@code MappedA}. */
   @ConfigPropertyName("mappy")
   Map<String, MappedA> mapStringMappedA();
+
+  /** Another map of string to {@code MappedA}, in its own section. */
+  @ConfigItem(section = "Section A")
+  @WithName("map.py")
```

Comment:
Is this scenario causing the `sts.sts` ?

Reply:
The issue was with `@ConfigItem(section = "sts")` : the section name was being appended to the property prefix, which is what we want for the file name, but not for displaying the property names.

the `@WithName` is unrelated, but since we don't have tests with that, I added one.